ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -9,6 +9,34 @@ <h1>Speaking Practice View</h1>
 <p id="text-for-timer"><span id="text-timer"> </span> </p>
 <progress id="timer-bar"></progress>
 
+<!-- original -->
```

Comment:
Furthermore, this should be wrapped in a form that points to the url  "{% url 'student:speaking_practice_submit course.id %}".

That form should probably be in a different file to keep things neat. See ComSemApp/templates/ComSemApp/student/attempt_form.html, ComSemApp/templates/ComSemApp/student/expression_list.html (line 38), and the student AttemptCreateView for an example of a form being pulled in from another file and used properly.

Reply:
This will be handled in #176